enterprise-suite/scripts/lbc.py

```diff
@@ -0,0 +1,404 @@
+                    # Pod name is the first thing on the line
+                    pod = line.split()[0]
+
+                    for container in get_pod_containers(pod):
+                        write_log(archive, pod, container)
+        else:
+            sys.exit(failure_msg + 'unable to pods in the namespace {}'.format(args.namespace))
+    
+    printerr('Lightbend Console diagnostic data written to {}'.format(filename))
+
+def setup_args():
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subcommand', help='sub-command help')
+
+    install = subparsers.add_parser('install', help='install es-console')
+    verify = subparsers.add_parser('verify', help='verify es-console installation')
+    diagnose = subparsers.add_parser('diagnose', help='make an archive with k8s status info for debugging and diagnostic purposes')
+
+    # Install arguments
+    install.add_argument('--dry-run', help='only print out the commands that will be executed',
+                        action='store_true')
+    install.add_argument('--force-install', help='set to true to delete an existing install first, instead of upgrading',
+                        action='store_true')
+    install.add_argument('--export-yaml', help='export resource yaml to stdout',
+                        choices=['creds', 'console'])
+    install.add_argument('--helm-name', help='helm release name', default='enterprise-suite')
+    install.add_argument('--local-chart', help='set to location of local chart tarball')
+    install.add_argument('--chart', help='chart name to install from the repository', default='enterprise-suite')
+    install.add_argument('--repo', help='helm chart repository', default='https://repo.lightbend.com/helm-charts')
+    install.add_argument('--version', help='es-console version to install', type=str)
+
+    install.add_argument('helm', help='arguments to be passed to helm', nargs=argparse.REMAINDER)
+
+    # Common arguments for all subparsers
+    for subparser in [install, verify, diagnose]:
+        subparser.add_argument('--skip-checks', help='skip environment checks',
+                            action='store_true')
+        subparser.add_argument('--creds', help='credentials file', default='~/.lightbend/commercial.credentials')
+        subparser.add_argument('--namespace', help='namespace to install es-console into/where it is installed', default='lightbend')
+
+    return parser.parse_args()
+
+def main():
+    global args
+    args = setup_args()
+
+    if args.subcommand == 'verify':
+        if not args.skip_checks:
+            check_kubectl()
+        check_install(args)
+    
+    if args.subcommand == 'install':
+        creds = import_credentials(args)
+
+        if not args.skip_checks:
+            if args.export_yaml == None:
+                minikube = is_running_minikube()
+                minishift = is_running_minishift()
+                preinstall_check(creds, minikube, minishift)
```

Comment:
Don't we still need to check if helm is installed when doing export_yaml?

Reply:
That's done in the else branch few lines below